packages/file_selector/file_selector_ios/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.5.2+1
+
+* Removes Swift Package Manager compatibility.
```

Comment:
How about "Temporarily remove Swift Package Manager compatibility to resolve issues with Cocoapods builds."?